<launch>
  <node pkg="freespace_mechanism_heuristic" type="move_base" respawn="false" name="compute_freespace_mechanism_heuristic" output="screen" required="true">
    <param name="footprint_padding" value="0.01" />
    <param name="controller_frequency" value="10.0" />
    <param name="controller_patience" value="100.0" />
    <param name="base_global_planner" value="gki_3dnav_planner/GKI3dNavPlanner" />
    <param name="GKI3dNavPlanner/motion_primitive_filename" value="$(find gki_3dnav_planner)/config/pr2_5cm_plussideways.mprim" />
    <param name="GKI3dNavPlanner/freespace_heuristic_costmap" value="$(find gki_3dnav_planner)/config/pr2_5cm_plussideways.mprim_costmap.dat" />

    <rosparam file="$(find gki_3dnav_planner)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find gki_3dnav_planner)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find gki_3dnav_planner)/config/local_costmap_params_close.yaml" command="load" />
    <rosparam file="$(find gki_3dnav_planner)/config/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find gki_3dnav_planner)/config/sbpl_global_params.yaml" command="load" />
    <rosparam file="$(find gki_3dnav_planner)/config/base_local_planner_params_close.yaml" command="load" />

    <rosparam file="$(find gki_3dnav_planner)/config/sbpl_local_params.yaml" command="load" />
    <rosparam file="$(find gki_3dnav_planner)/config/base_local_planner_poses.yaml" command="load" />

    <remap from="cmd_vel" to="navigation/cmd_vel"/>
  </node>
</launch>
